minibear/src/main/java/frc/robot/NavX/ContinuousAngleTracker.java

```diff
@@ -0,0 +1,122 @@
+package frc.robot.NavX;
+
+class ContinuousAngleTracker {
+
+    private boolean fFirstUse;
+    private double gyro_prevVal;
+    private int ctrRollOver;
+    float curr_yaw_angle;
+    float last_yaw_angle;
+    double angleAdjust;
+    
+    public ContinuousAngleTracker() {
+    	init();
+        angleAdjust = 0.0f;    	
+    }    
+
+    public void init() {
+        gyro_prevVal = 0.0;
+        ctrRollOver  = 0;
+        fFirstUse = true;
+        last_yaw_angle = 0.0f;        
+        curr_yaw_angle = 0.0f;
+    }
+    
+    public void nextAngle( float newAngle ) {
+		synchronized(this){
+			last_yaw_angle = curr_yaw_angle;
+			curr_yaw_angle = newAngle;
+		}
```

Comment:
Whitespace is funky in this file.  Someone should make sure it is cleaner for readability.